src/main/resources/api/openapi.yaml

```diff
@@ -1718,10 +1716,12 @@ components:
             - $ref: '#/components/schemas/Timestamp'
         headersScore:
           type: integer
+          format: int64
```

Comment:
could be out of 64 bits, so better to stay on format being skipped I guess

Reply:
Removed & added a note to description